recipes/fs/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "fs" %}
+{% set version = "2.0.9" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "8570c6349db3b0bf68fe615fe2683ca04796082aceae0ae1d232f71a6f6e47ef" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - appdirs >=1.4.3,<2
+    - pytz
+    - six >=1.10.0,<2
+    - enum34 >=1.1.6,<2  # [py2k]
+    - scandir >=1.5,<2  # [py2k or py34]
+    
+  run:
+    - python
+    - appdirs >=1.4.3,<2
+    - pytz
+    - six >=1.10.0,<2
+    - enum34 >=1.1.6,<2  # [py2k]
+    - scandir >=1.5,<2  # [py2k or py34]
+
+test:
+  requires:
+    - setuptools
+  imports:
+    - fs
+    - fs.opener
+
+about:
+  home: https://www.pyfilesystem.org/
+  license: MIT
+  license_family: MIT
+  summary: Filesystem abstraction layer for Python
+  description: "pyfilesystem2 lets you work with diverse filesystems through a common API."
+  doc_url: 'http://pyfilesystem2.readthedocs.io/en/stable/'
```

Comment:
That will work, but the canonical docs url is `https://docs.pyfilesystem.org/en/latest/` now.

Reply:
conda-forge guidelines insist on linking to specific version, not to "latest": http://conda-forge.readthedocs.io/en/latest/guidelines.html?highlight=latest

Reply:
Ah, ok. The current latest version is https://docs.pyfilesystem.org/en/v2.0.9/